tests/scripts/azp-strategy.ps1

```diff
@@ -51,11 +51,11 @@ $terraformVersionsCount = $terraformVersions.Count
 
 #######################################
 # Terraform AzureRM Provider Versions
-# - Base Version: (3.74.0)
+# - Base Version: 3.107.0)
```

Comment:
Bracket missing. This one is just to prove I am actually reviewing the code. :)

Reply:
Fixed it - thanks !